templates/_composition/_shared/Feature.ShareTarget/Activation/ShareTargetActivationHandler.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+using System.Threading.Tasks;
+using Windows.ApplicationModel.Activation;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Param_ItemNamespace.Views;
+
+namespace Param_ItemNamespace.Activation
+{
+    internal class ShareTargetActivationHandler : ActivationHandler<ShareTargetActivatedEventArgs>
+    {
+        protected override async Task HandleInternalAsync(ShareTargetActivatedEventArgs args)
+        {
+            var frame = new Frame();
```

Comment:
This will have issues in Caliburn.Micro as you're wiping out the `Frame` being wrapped by the navigation service which has a lot of flow on effects around things like lifecycle.